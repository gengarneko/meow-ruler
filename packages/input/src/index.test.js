import React from 'react'
import renderer from 'react-test-renderer'
import '@emotion/jest'

import Input from '.'

describe('Input', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Input />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
