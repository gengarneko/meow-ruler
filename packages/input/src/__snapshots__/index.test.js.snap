// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders correctly 1`] = `
<input
  className="css-beq8e8-Input e1q4f7if0"
/>
`;
